eftest/src/eftest/runner.clj

```diff
@@ -72,7 +81,7 @@
         (if (:multithread? opts true)
           (let [test (bound-fn* test-var)]
             (dorun (->> vars (filter synchronized?) (map test)))
-            (dorun (->> vars (remove synchronized?) (map (fn [v] #(test v))) run-in-parallel)))
+            (dorun (->> vars (remove synchronized?) (map (fn [v] #(test v))) pcalls*)))
```

Comment:
This line could be changed to:

```clojure
(dorun (->> vars (remove synchronized?) (pmap* test)))
```

I think?

Reply:
agree!